.github/workflows/production-tag.yml

```diff
@@ -0,0 +1,128 @@
+name: Create Production Tag
+
```

Comment:
This based on the Content Build Daily production release (tag), with some (IMHO) improvements and changes. 

https://github.com/department-of-veterans-affairs/content-build/actions/runs/7506884417/workflow